components/apimgt/org.wso2.carbon.apimgt.rest.api/src/main/java/org/wso2/carbon/apimgt/rest/api/RestApiConstants.java

```diff
@@ -24,4 +24,7 @@
     public static final String APPLICATION_JSON = "application/json";
 
     public static final String API_ID_DELIMITER = "-";
+
```

Comment:
Fixed in #1581(https://github.com/wso2/carbon-apimgt/commit/73c207a285e09dde2475ebe8182da3532a01431c)